### ISSUE TITLE
Error Importing Teslafi Logs

### ISSUE THREAD

**Issue by Vipercat** [adriankumpf/teslamate#820]
**Describe the bug**

I receive an error when trying to import the Teslafi.csv logs. The logs were generated by using the "tesla-apiscraper-to-teslafi-export" repository. 

**Expected behavior**

I would expect the logs to be imported succesfully

**How to reproduce it (as minimally and precisely as possible)**:

1. stop the teslamate service (in my case the "docker_teslamate_1" container'
2. copy the csv files to the import folder
3. start the teslamate service
4. Goto http://127.0.0.1:4000 and
5. I'm presented with the import screen (http://127.0.0.1:4000/import) and all 4 files that I copied into the import-folder are shown
6. Timezone is selected properly
7. Click on 'import'. Error in log

**Relevant entries from the logs**

2020-07-05 11:49:06 stdout 13:49:06.050 [info] Sent 200 in 25ms 
2020-07-05 11:49:28 stdout 13:49:28.737 [info] Importing 4 file(s) ... 
2020-07-05 11:49:28 stdout 13:49:28.799 [info] Starting logger for '<NAME OF MY CAR>' 
2020-07-05 11:49:28 stdout 13:49:28.803 [error] GenStateMachine TeslaMate.Import terminating 
2020-07-05 11:49:28 stdout ** (MatchError) no match of right hand side value: {:error, #Ecto.Changeset<action: :update, changes: %{eid: 32524}, errors: [vid: {"is invalid", [type: :integer, validation: :cast]}], data: #TeslaMate.Log.Car<>, valid?: false>} 
2020-07-05 11:49:28 stdout (teslamate 1.19.4) lib/teslamate/vehicles.ex:102: TeslaMate.Vehicles.create_or_update!/1 
2020-07-05 11:49:28 stdout (teslamate 1.19.4) lib/teslamate/import.ex:126: TeslaMate.Import.handle_event/4 
2020-07-05 11:49:28 stdout (stdlib 3.12.1) gen_statem.erl:1120: :gen_statem.loop_state_callback/11 
2020-07-05 11:49:28 stdout (stdlib 3.12.1) proc_lib.erl:249: :proc_lib.init_p_do_apply/3 

**Environment**

- TeslaMate version: 1.19.4
- Type of installation: Docker on Synology
- OS TeslaMate is installed on: Linux DiskStation 4.4.59+ #25426 SMP PREEMPT Tue May 12 04:54:42 CST 2020 x86_64 GNU/Linux synology_denverton_1618+ 
- User OS & Browser: Windows 10 (v2004) & Edge Browser
- Others:

**Comment by Vipercat** [comment]
Problem Solved!

The "**tesla-apiscraper-to-teslafi-export**" repository tool populated the Teslafi_xyyy_.csv files with the wrong **vehicle_id**, **id** and **id_s** values. By querying the car myself with a API tool I was able to get the correct values and replace them in the .csv files.

**Comment by adriankumpf** [comment]
Then you were apparently one of the first users who tried the export tool :) @olexs would certainly be interested to know about this bug to fix it. It would probably be a good idea to create an issue here: https://github.com/olexs/tesla-apiscraper-to-teslafi-export?

**Comment by Vipercat** [comment]
@adriankumpf, I caused the issue myself it seems. At a certain stage in the process, the export tool asks the user for the vehicle ID and instead of entering the vehicle ID I entered the VIN. My mistake in that regard!
On the other hand, I had no idea what the vehicle ID of my car was and therefore thought the VIN was asked. Only after querying the car via API I noticed the "vehicle_id" field and 1 and 1 was 2. Then I was struggling with this a bit longer because my vehicle ID showed as 9 digits whereas I noticed that others users had reported a vehicle ID of 10 digits. The same was reported on the this site: https://www.teslaapi.io/

Another issue I encountered was the numbers the tool generated for the 'id' and 'id_s' fields. Those were all 9's instead of the actual numbers that I was able to query for these fields. Nevertheless, in the end all went fine and the process of exporting from tesla_apiscraper, converting to teslafi.csv, correcting the .CSV with the correct ID's and importing in Teslamate was pretty simple and flawless.

Thanks again for this great solution!
